sync-docs.sh

```diff
@@ -17,7 +17,7 @@ function replace() {
 
 function main() {
     if [ "$#" -eq 0 ]; then
-	SDKS="dotnet go java js py hs"
+	SDKS="dotnet go java js py hs cpp"
```

Comment:
So - the C++ docs do *not* have an `API.md` in the docs directory as it's a doxygen.

If you look at lines 42 and 43 we curl for both `API.md` and `README.md`

so this will introduce a guaranteed 404 and delay in builds for the C++ guide.

We also are not (at the point of this PR) creating a standalone C++ page (we can do that on a second pass along with more `case esac` statements in bash.

We can either:

1. Revert this change and come back to it later , or
2. Create `/developers/cpp/minio-cpp.rst`
3. Include `/developers/cpp/quickstart.md`
4. Not have a TOC for that page

You can look at `minio-java.rst` for an idea of how that would owrk.

Reply:
I see your point and you are right, I will revert this change for later.